website/content/docs/k8s/operations/uninstall.mdx

```diff
@@ -9,70 +9,70 @@ description: Uninstall Consul on Kubernetes
 Uninstalling Consul requires running `helm delete` **and** then manually cleaning
 up some resources that Helm does not delete.
 
-1. First, run `helm delete`:
+  1. First, run `helm delete`:
 
-   ```shell-session
-   $ helm delete hashicorp
-   release "hashicorp" uninstalled
-   ```
+  ```shell-session
+  $ helm delete hashicorp
+  release "hashicorp" uninstalled
+  ```
 
-1. After deleting the Helm release, you need to delete the `PersistentVolumeClaim`'s
-   for the persistent volumes that store Consul's data. These are not deleted by Helm due to a [bug](https://github.com/helm/helm/issues/5156).
-   To delete, run:
+  2. After deleting the Helm release, you need to delete the `PersistentVolumeClaim`'s
+  for the persistent volumes that store Consul's data. These are not deleted by Helm due to a [bug](https://github.com/helm/helm/issues/5156).
+  To delete, run:
 
-   ```shell-session
-   $ kubectl get pvc -l chart=consul-helm
-   NAME                                   STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
-   data-default-hashicorp-consul-server-0   Bound    pvc-32cb296b-1213-11ea-b6f0-42010a8001db   10Gi       RWO            standard       17m
-   data-default-hashicorp-consul-server-1   Bound    pvc-32d79919-1213-11ea-b6f0-42010a8001db   10Gi       RWO            standard       17m
-   data-default-hashicorp-consul-server-2   Bound    pvc-331581ea-1213-11ea-b6f0-42010a8001db   10Gi       RWO            standard       17m
+  ```shell-session
+  $ kubectl get pvc -l chart=consul-helm
+  NAME                                   STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+  data-default-hashicorp-consul-server-0   Bound    pvc-32cb296b-1213-11ea-b6f0-42010a8001db   10Gi       RWO            standard       17m
+  data-default-hashicorp-consul-server-1   Bound    pvc-32d79919-1213-11ea-b6f0-42010a8001db   10Gi       RWO            standard       17m
+  data-default-hashicorp-consul-server-2   Bound    pvc-331581ea-1213-11ea-b6f0-42010a8001db   10Gi       RWO            standard       17m
 
-   $ kubectl delete pvc -l chart=consul-helm
-   persistentvolumeclaim "data-default-hashicorp-consul-server-0" deleted
-   persistentvolumeclaim "data-default-hashicorp-consul-server-1" deleted
-   persistentvolumeclaim "data-default-hashicorp-consul-server-2" deleted
-   ```
+  $ kubectl delete pvc -l chart=consul-helm
+  persistentvolumeclaim "data-default-hashicorp-consul-server-0" deleted
+  persistentvolumeclaim "data-default-hashicorp-consul-server-1" deleted
+  persistentvolumeclaim "data-default-hashicorp-consul-server-2" deleted
+  ```
 
    ~> **NOTE:** This will delete **all** data stored in Consul and it can't be
    recovered unless you've taken other backups.
```

Comment:
```suggestion
~> **NOTE:** This will delete **all** data stored in Consul and it can't be
   recovered unless you've taken other backups.
```

I think we need to indent this to fix the rendering.

![image](https://user-images.githubusercontent.com/29551334/133466461-b6af1d1e-27d6-42d6-bc72-d8b218aa0519.png)